iphone/Maps/Classes/MapDownloader/MWMBaseMapDownloaderViewController.mm

```diff
@@ -501,6 +501,11 @@ - (IBAction)longPress:(UILongPressGestureRecognizer *)sender
 
 - (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
 {
+  if (buttonIndex >= actionSheet.numberOfButtons)
```

Comment:
Но комментарий-то нужно вставить в любом случае.

> On Apr 8, 2016, at 16:16, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/MapDownloader/MWMBaseMapDownloaderViewController.mm <https://github.com/mapsme/omim/pull/2779#discussion_r59022474>:
> 
> > @@ -501,6 +501,11 @@ - (IBAction)longPress:(UILongPressGestureRecognizer *)sender
> >  
> >  - (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
> >  {
> > +  if (buttonIndex >= actionSheet.numberOfButtons)
> Вероятность бага в системной апишке на порядки ниже вероятности бага в нашем коде.
> 
> да, но в этом случае сработало более редкое событие.
> по поводу того, как исправлять: Илья посчитал, что в тех условиях, когда делался конкретно этот пр, такого фикса будет достаточно. я посчитал так же.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2779/files/362adb4fcf0abf0f56f27377d9292e1ce953f00f#r59022474>